lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -267,10 +276,6 @@ putTokenTransfer _ (Tx inputs outputs) = do
     putOutput (TxOut address coin) = do
         putAddress address
         putWord64be $ getCoin coin
-    putAddress address = do
-        -- NOTE: only single address supported for now
-        putWord8 (single @n)
```

Comment:
yes - I was not happy to introduce new constrain just to resolve the network here. Having this bit in address is much more elegant solution